src/app/shared/palette/money-gbp/money-gbp.builder.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { CaseField } from '../../domain/definition/case-field.model';
+
+@Injectable()
+export class MoneyGBPCaseFieldBuilder {
```

Comment:
wonder whether we could have a more classic approach to define a MoneyGBPCaseField which is a subclass of CaseField. 
then you could just create it with new MoneyGBPCaseField(value) anywhere you need that